model_compression_toolkit/qunatizers_infrastructure/pytorch/base_pytorch_quantizer.py

```diff
@@ -40,6 +40,14 @@ def __init__(self,
             super().__init__(quantization_config, quantization_target, quantization_method)
 
 
+        def initialize(self):
```

Comment:
Why add an initialization method?

Reply:
This is because the base init arguments seems to be relevant for keras. Let's talk in the infra meeting.